fizz-buzz-streams/src/main/java/com/pubsap/eng/streams/Main.java

```diff
@@ -63,7 +63,9 @@ public static Topology buildTopology(Config config,
                         Named.as("the-grouped-topic"),
                         Materialized.with(inputKeySerde, outputSerde)
 
-                ).toStream()
+                )
```

Comment:
Why ?? 🤷‍♂

Reply:
For elegance 🤴 and to expose the type after the aggregate function like the others operations on the topography:
<img width="505" alt="Capture d’écran 2020-02-17 à 12 46 48" src="https://user-images.githubusercontent.com/6732011/74651241-a93af380-5183-11ea-8464-4ac59b10b0c7.png">

Reply:
smart